api/types.go

```diff
@@ -19,6 +24,15 @@ type Hypothesis struct {
 }
 
 type Responder struct {
+	ID         uuid.UUID
+	Properties types.JSONStringMap `json:"properties" gorm:"type:jsonstringmap;<-:false"`
+	TeamID     uuid.UUID
+	// TODO: Impement Scan+Value for kommons.EnvVar for gorm to work correctly
```

Comment:
@moshloop Has this been implemented somewhere or should I create an issue in the kommons repo and add this there ?

Basically, for gorm to use kommons.EnvVar for storage and query we need to implement methods similar to jsonStringMap

Reply:
So we wouldn;t be persisting envVar to the db, we are marshalling the entire struct and then saving it as json